decidim-core/config/locales/en.yml

```diff
@@ -1483,7 +1486,6 @@ en:
         no_conversations_warning: There are no conversations yet.
         no_results: No results
         participant_with_disabled_message_reception: This participant has disabled message reception
-        time_ago: "%{time} ago"
```

Comment:
Same as above, I would rather leave this existing translation than introducing it under a new key to avoid labeling this as change.